scripts/update_database.py

```
@@ -97,7 +101,25 @@ def update_database():
+            # Check if books table has notes_shared column
+            try:
+                books_columns = inspector.get_columns('books')
+                books_col_names = [col['name'] for col in books_columns]
+
+                if 'notes_shared' not in books_col_names:
+                    logger.info("  [+] Adding missing column: notes_shared to books table")
+                    sql = text('ALTER TABLE books ADD COLUMN notes_shared BOOLEAN DEFAULT 0 NOT NULL')
+                    db.session.execute(sql)
+                    db.session.commit()
+                    logger.info("      -> Added successfully.")
+            except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)